.travis.yml

```diff
@@ -1,4 +1,14 @@
 language: node_js
 node_js:
   - "12"
-script: npm run build && npm test
+install:
+   - npm install -g markdownlint-cli
```

Comment:
Travis-ci is failing because you are overwriting the install step, you also need `npm install` 😄

Reply:
Yeah I saw that, I will investigate the error message.

Reply:
Hmmm I cannot see that I overwrote an install step from the _diff_, is there an implicit install step?